src/lib/utils.js

```diff
@@ -57,8 +57,10 @@ export const trimAndFormatPath = (
   );
 };
 
-// $FlowFixMe
-export const getTerminalWidth = (): number => process.stdout.columns;
+export const getTerminalWidth = (
+  pipe: stream$Writable | tty$WriteStream = process.stdout,
```

Comment:
bug fix?

Reply:
Nop, not a bug fix... It was failing in CI because columns is 0. I changed so that it can use the pipe that was passed as an argument, which means that now we can mock the number of columns.